main/conceptual/js-programming/far.md

```diff
@@ -1,7 +1,7 @@
 
-# `Far()`, Remotable, and Marshaling
+# Far(), Remotable, and Marshaling
```

Comment:
So we're getting rid of code markup in headings? Does it make sense to keep the ()s, then?

Reply:
I was still working out what formatting looked right when I got interrupted by Rowland's request. (I really didn't like the code markup in the heading; it wasn't very readable & looked weird.)

So... yeah. I'll come back to this in the near future & apply a consistent formatting throughout the docs.